crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1685,7 +1691,7 @@ impl TransportRpcModules {
     /// Returns [Ok(false)] if no ipc transport is configured.
     pub fn merge_ipc(&mut self, other: impl Into<Methods>) -> Result<bool, RegisterMethodError> {
         if let Some(ref mut ipc) = self.ipc {
-            return ipc.merge(other.into()).map(|_| true)
+            return ipc.merge(other.into()).map(|_| true);
```

Comment:
```suggestion
            return ipc.merge(other.into()).map(|_| true)
```

Reply:
Got rugpulled by rustfmt being trigger-happy.
Undone in 3e5f678f8431096f36ec7185de038b7a72b2b0c4